test/integration/StreamrClient.test.js

```diff
@@ -16,7 +16,9 @@ describe('StreamrClient', () => {
     const createClient = (opts = {}) => new StreamrClient({
         url: config.websocketUrl,
         restUrl: config.restUrl,
-        apiKey: 'tester1-api-key',
+        auth: {
+            privateKey: '0x12345564d427a3311b6536bbcff9390d69395b06ed6c486954e971d960fe8709',
```

Comment:
Side note: unlike addresses, the private key is often expressed without `0x` prefix. I wonder if that works too?